packages/discord-utilities/src/lib/regexes.ts

```diff
@@ -29,10 +29,10 @@ export const DiscordHostnameRegex =
 
 /**
  * Regex that can can capture the code of Discord invite links
- * @raw `/^(?:https?:\/\/)?(?:www\.)?(?:discord\.gg\/|discord(?:app)?\.com\/invite\/)?(?<code>[\w\d-]{2,})$/i`
+ * @raw `/discord(?:(?:app)?\.com\/invite|\.gg(?:\/invite)?)\/(?<code>[\w-]{2,255})/gi`
  * @remark Capture group 1 is the invite URL's unique code. It is named `code`.
  */
-export const DiscordInviteLinkRegex = /^(?:https?:\/\/)?(?:www\.)?(?:discord\.gg\/|discord(?:app)?\.com\/invite\/)?(?<code>[\w\d-]{2,})$/i;
+export const DiscordInviteLinkRegex = /discord(?:(?:app)?\.com\/invite|\.gg(?:\/invite)?)\/(?<code>[\w-]{2,255})/gi;
```

Comment:
We probably want to add `(?:^|\b)` at the start and `(?:$|\b)` at the end

Reply:
Check my message in discord. You can do like `"discord.gg/invite"`

Reply:
### Looks like @vladfrangu's way works good @imranbarbhuiya 
![image](https://user-images.githubusercontent.com/85353424/170170663-75ea038f-efa5-4ecb-a017-b33a9bf87eec.png)